docs/src/design/decoder/main.md

```diff
@@ -134,14 +136,14 @@ We denote the running product column used to keep track of hash chiplet state as
 To initiate a 2-to-1 hash of $8$ elements ($v_0, ..., v_7$) we need to divide $p_0$ by the following value:
 
 $$
-\alpha_0 + \alpha_1 \cdot m_{bp} + \alpha_2 \cdot r + \alpha_4 \cdot 8 + \sum_{i=0}^7 (\alpha_{i+8} \cdot v_i)
+\alpha_0 + \alpha_1 \cdot m_{bp} + \alpha_2 \cdot r + f_{ctrlb} \cdot \alpha_5 \cdot \sum_{b=0}^6(b_i \cdot 2^i) + \sum_{i=0}^7 (\alpha_{i+8} \cdot v_i)
 $$
 
 where:
 * $m_{bp}$ is a label indicating beginning of a new permutation. Value of this label is computed based on hash chiplet selector flags according to the methodology described [here](../chiplets/hasher.md#multiset-check-constraints).
+* $f_{ctrlb}$ is a flag indicating whether the current operation is a control block initializer (excluding span).
 * $r$ is the address of the row at which the hashing begins.
-* $\alpha_4 \cdot 8$ indicates that we are hashing a total of $8$ elements.
-* Some $\alpha$ values are skipped in the above (e.g., $\alpha_3$) because of the specifics of how auxiliary hasher table rows are reduced to field elements (described [here](../chiplets/hasher.md#multiset-check-constraints)). For example, $a_3$ is used a coefficient for node index values during Merkle path computations in the hasher, and thus, is not relevant in this case.
+* Some $\alpha$ values are skipped in the above (e.g., $\alpha_3$) because of the specifics of how auxiliary hasher table rows are reduced to field elements (described [here](../chiplets/hasher.md#multiset-check-constraints)). For example, $\alpha_3$ is used a coefficient for node index values during Merkle path computations in the hasher, and thus, is not relevant in this case.  The $\alpha_4$ is omitted when the number of items being hashed is a multiple of the rate width ($8$) because it is multiplied by 0 - the value of the first capacity register as determined by the [hasher chiplet logic](../chiplets/hasher.md#simple-2-to-1-hash).
```

Comment:
nit: missing word - "For example, $\alpha_3$ is used as a coefficient..."

Reply:
fixed